frontend/index.html

```diff
@@ -1 +1,24 @@
-<!-- O FRONTEND SERÁ DESENVOLVIDO DENTRO DESTE DIRETORIO -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pokédex</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="pkHeader">Pokédex</h1>
+    <div class="caixa">
+        <label for="pkInput">Digite o nome ou número do Pokémon:</label>
+        <input type="text" id="pkInput" placeholder="Nome ou número">
+        <button onclick="fetchPokemon()">Buscar</button>
+    <div id="pkInfo" class="pokemon-info">
+            <img id="pkImage" src="./assets/pokemon.png" alt="Imagem do Pokémon" class="hidden">
```

Comment:
A imagem em questão do pokemon vai ser puxada pelo backend, porém quando eu for fazer as funções para chamada da api, ja altero essa linha.